src/main/java/types/TpmSupport.java

```diff
@@ -0,0 +1,10 @@
+package types;
+
+import org.ovirt.api.metamodel.annotations.Type;
+
+@Type
+public enum TpmSupport {
+    UNSUPPORTED,
+    SUPPORTED,
+    REQUIRED;
```

Comment:
Could you add some documentation to explain what each item means? It's almost straight-forward but still there is some room for confusion (especially between supported and required)

Reply:
@rszwajko Yes, this is correct.